Podfile

```diff
@@ -7,7 +7,6 @@ target 'SEDaily-IOS' do
 
   # Pods for SEDaily-IOS
   pod 'Alamofire', '~> 4.0’
```

Comment:
Minor fix here, the last single quote should be ' not ’. I got a cocoapod warning about this.